docs/docs/contributing/workflow.md

```diff
@@ -78,7 +50,7 @@ scala> import dotty.tools.DottyTypeStealer._; import dotty.tools.dotc.core._; im
 
 Now, you can define types and access their representation. For example:
 
-```scala
+```bash
```

Comment:
I'm pretty sure this was to get nicer syntax-highlighting. But I'm not sure I'm the one who wrote it..